spec/support/requests_spec_helper.rb

```diff
@@ -20,6 +20,6 @@ def bypass_rbac
   end
 
   def catalog_admin_role
-    Catalog::RBAC::Role::ADMINISTRATOR_ROLE_NAME
+    'Catalog Administrator'
```

Comment:
Why are we dropping the constant?  Is this because we really should not be check roles anymore?

Reply:
@gmcculloug lib/catalog/rbac/role.rb has been deleted

Reply:
Well, I removed the `Catalog::RBAC::Role` class, so, we could make it a constant here I guess. Eventually this will go away entirely anyway.